src/resources/swagger.yaml

```diff
@@ -1894,3 +1911,13 @@ components:
         donorsUnchangedCount:
           description: 'Amount of donors left unchanged.'
           type: integer
+
+    MissingEntityListResponse:
+      type: array
+      items:
+        type: object
+        properties:
+          programId:
+            type: string
+          donorCount:
+            type: integer
```

Comment:
This object can be added as a MissingEntityException schema, so that the entire response here can easily be written as an array of ref: MissingEntityException.

Might as well refactor this way since the return type for the GET request is a MissingEntityException